material-ui-superselectfield/material-ui-superselectfield.d.ts

```diff
@@ -0,0 +1,45 @@
+declare module "material-ui-superselectfield" {
+    import * as React from "react"
+
+    interface IMaterialUISuperSelectFieldProps {
+        name: string;
+        value: object[] | object | null;
+        multiple?: boolean;
+        key?: string;
+        onChange?: (...args: any[]) => void;
+        selectionsRenderer?: (...args: any[]) => void;
+        checkPosition?: string;
+        hintText?: string;
+        nb2show?: number;
+        elementHeight?: number;
+        style?: Object;
+        innerDivStyle?: Object;
+        selectedMenuItemStyle?: Object;
+        hoverColor?: string;
+        floatingLabel?: any;
+        hintTextAutocomplete?: string;
+        noMatchFound?: string;
+        anchorOrigin?: { vertical: string, horizontal: string };
```

Comment:
Left `anchorOrigin` with `horizontal` and `vertical`, as these are the only key:value pairs allowed according to the docs.